utils/evaluate/evaluate.go

```diff
@@ -45,7 +45,13 @@ func EvaluateResult(result interface{}, metric v1alpha1.Metric, logCtx logrus.En
 	}
 
 	if failCondition {
-		return v1alpha1.AnalysisPhaseFailed, nil
+		if metric.SuccessCondition != "" {
```

Comment:
I think it's possible to have a success and fail condition for 1 metric?  so wouldn't fail always win, even if it was the success condition that didn't pass?